src/radiation/radiation.cpp

```diff
@@ -289,25 +243,13 @@ void ablate::radiation::Radiation::Initialize(const solver::Range& cellRange, ab
                  * It doesn't matter which method is used,
                  * this will be the same procedure.
                  * */
-                switch (dim) {
```

Comment:
Now that the UpdateCoordinates function has an advancement parameter, the particle stepping can be replaced with that function call. (The advancement is necessary to avoid confusing the swarm by placing a particle directly on a cell face)